internal/server/biopage/bio_page.go

```diff
@@ -55,8 +55,10 @@ func GetBioPageData(
 	if err != nil {
 		return nil, err
 	}
-	if _, ok := data[0][dcid]; !ok {
-		return nil, nil
+	for _, baseData := range baseDataList {
```

Comment:
Perhaps add a comment that there is no merging going on here, and order is based on the preferred base-cache list?

Reply:
Done